docs/tutorials/timeseries/forecasting-model-zoo.md

```diff
@@ -181,28 +181,8 @@ Note that some of the models' hyperparameters have names and default values that
 
 ## MXNet Models
 
-Following MXNet-based models from GluonTS are available in AutoGluon.
+MXNet models from GluonTS have been deprecated as of AutoGluon v0.9 because of dependency conflicts caused by MXNet.
```

Comment:
I think we will skip 0.9 and do 1.0 release directly this year, @Innixma can confirm

Reply:
Fixed 👍